Gopkg.toml

```diff
@@ -95,15 +95,16 @@ required = [
   name = "golang.org/x/oauth2"
   revision = "cdc340f7c179dbbfa4afd43b7614e8fcadde4269"
 
-# TODO why is this overridden?
+# Select the version of knative/pkg we would like to import.
+# This controls when we upgrade apis.
 [[override]]
   name = "github.com/knative/pkg"
-  # HEAD as of 2019-03-12
-  revision = "684620ff81ae137be5cda634a03831b373adc4eb"
+  # HEAD as of 2019-03-18
+  revision = "587fb819f44ff27a38a8ceb172058c5214a47911"
 
 [[constraint]]
   name = "github.com/knative/serving"
-  version = "v0.3.0"
+  version = "v0.4.0"
```

Comment:
This needs to be mentioned in release notes.

Reply:
Added

Reply:
Can you add a reason why the update occurred? Even if it's just "because of a dependency conflict". Then users can see that no new functionality was added, there was no breaking change, etc.